daemon/src/db.rs

```diff
@@ -296,8 +306,8 @@ pub async fn load_cfd_by_order_id(
     let max_quantity = serde_json::from_str(row.max_quantity.as_str()).unwrap();
```

Comment:
Parsing `max_quantity` via `serde` here is not going to work is it? We are only `to_string`ing it on the insert.

Reply:
FYI, all things that involve `serde_json::from_str()` will go away with one exception:

```rust
    // TODO:
    // still have the use of serde_json::from_str() here, which will be dealt with
    // via https://github.com/comit-network/hermes/issues/290
    Ok(Cfd {
        order,
        quantity_usd: Usd(Decimal::from_str(&row.quantity_usd).unwrap()),
        state: serde_json::from_str(row.state.as_str()).unwrap(),
    })
```